react/lib/tests/util/api-client.test.ts

```diff
@@ -0,0 +1,148 @@
+import { shouldTriggerOnSuccess } from '../../util/api-client';
+import { Transaction, resolveNumber } from '../../util';
+
+jest.mock('axios');
+
+global.fetch = jest.fn();
+
+describe('API Client Util Tests', () => {
+  describe('shouldTriggerOnSuccess', () => {
+    it('returns true when all conditions match', () => {
```

Comment:
These test names should make sense on their own. This one is super generic and doesn't really tell me anything.